src/Compilers/CSharp/Test/CommandLine/project.json

```diff
@@ -1,5 +1,7 @@
 ﻿{
   "dependencies": {
+    "xunit": "2.1.0",
```

Comment:
@jasonmalinowski for this change I just mapped an reference in csproj to an entry in project.json.  I didn't really give a ton of thought to whether or not we should have essentially redundant references.  I wouldn't be opposed to a policy here and really it should be fairly easy to enforce.